index.html

```diff
@@ -14,22 +14,30 @@
 </head>
 
 <body>
-    <header class="flex">
-        <div class="nav-bar flex">
+    <header class="">
```

Comment:
- [ ] OPTIONAL: I would recommend to bug the project and make it responsive. I have noticed that you fixed width everywhere in mobile view to `375px` this is introducing a number of issues. For instance, mobile screens can have different width, from `320` to `425`, but your  application renders properly only for certain width which is a wrong approach. Most of the time users will have to scroll the body left and right to see the content.

https://user-images.githubusercontent.com/77038610/161607722-1af914d9-1973-42a5-b449-fe2b92053003.mp4

So, when you get time take try to make your portfolio responsive 😉